packages/rum-core/test/performance-monitoring/transaction-service.spec.js

```diff
@@ -245,23 +243,12 @@ describe('TransactionService', function() {
   })
 
   it('should capture resources from navigation timing', function(done) {
-    var _getEntriesByType = window.performance.getEntriesByType
-
-    window.performance.getEntriesByType = function(type) {
```

Comment:
Why we these need to be removed? since TransactionService make use of capture navigation timing

Reply:
test basically doesn't do any checks on the spans, just patching the window function and returning some data for no reason is not useful IMO. Let me put this way, why are there entries useful here?

Reply:
The test is "should capture resources from navigation timing" in other words we want to test whether TransactionService captures navigation timings and for that to work we need the entries! Maybe we should addd extra checks here to make sure navigation timings are captured correctly.

Reply:
I will add more checks to the test then to make sure we are actually capturing.